libraries/Microsoft.Bot.Builder.Runtime/CoreBot.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Claims;
+using System.Security.Principal;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Bot.Builder.Runtime.Settings;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Bot.Builder.Runtime
+{
+    public class CoreBot : ActivityHandler
```

Comment:
Looking at all other examples I can find across this repo in the tests, as well as in BotFramework-Composer and botframework-solutions, the trend appears to be that they all inherit from ActivityHandler. I don't have as much intimate knowledge of how it's utilized throughout the SDK, but it would matter if we were dependent upon the ActivityHandler class and not IBot, so I'm inclined to stick to the pattern unless there's a good or clear reason not to.

Reply:
I believe the consensus was that it should implement IBot, and not inherit from ActivityHandler.  Correct?

Reply:
Yes IBot, at least until we have a unification story for ActivityHandler and DialogManager.